utils/wizcloudProccess/getAll/bankPageRecords.ts

```diff
@@ -0,0 +1,16 @@
f3443a7618ebaa7a8266c8d6766cb9404408f225df330b58ffc7c6c1cf736e3db5728d3aeacf6f66bd08455e921ff97381c7b3d458b2a2ec98f924a228da2bec6071783a475dc08794b1210f51c92861a568d7066b46ccbbcd2568961d18b7402284876190e47acf6122eabc1e97f00239fdf3b5a14dd428b1262065034a6379ca7236799ae7aaaee8ac32f3b680bafd6fc3c53119c5b1289167d4a72262fc8b77502d608d0df6d1be6c3f2442978c0be92b1594bf2c054fc8d3136a3911d0be230709113774706bf2dc480278747b668e5a025a8f283854e256f637f3573e61afbf8222c9f9bf1244d42d7e99de597d1923036cec5f75d11b346158ab6b51178090426244578c24b68ecd16167620dbe007b1df4225ed97f251adb8d19be5572f477fffef0cd614d5d29ead87d79bda0a1e1c1ae837e3c0201ddc69e80bc2614b90990c3ba950b9de172e2af626017e21c6efaf29fd6f512e4a5a7a50fd8132e733f1363acbf10be55abdd5d30a3cb1f3842ff9fc1b7c8528eac7a884a94612440d673506a2e405dd04908a34b6743cf17a98c5a6a42d1d8c5bd6c1c1918c7590ff5483fdbb1b485a2f3409b2aae1aee1bf7df3c5bc5da5bc31356880ba9abafa01f438b3a9f32283806b3052b35714ddc7ef907472dc8f1cfc08d55eb2a1b4579c699488183965e5061582cb376218d6291149e03cd8d45d9416b939236fa3853bb740daff224debf08ddba809476b2f63e656863dd22a36108cbb08d536719640ff34874dbda0b2ccb98050fde8d61f0bb7d09fc7310fce0b359dc68db2cf3550ffc72b64bc449035b71831bcb47ec29bbf4c6631e4786d15cf824ae3ded9fb0d56e87d24d8bb1b4fb01e6a8c64954433539d6bcca588235e3dd1a512cf4051d7b4cf6d5725358b90ad7dbed20d8cbab08ef34e6b1e39d4fcbd54c6d965fb0984762cfbfe366c7e2919e74ba7db2c5802bd6d8e03a6c8f0d9ade12d7363da69040e1dd7ecde10193cb7a643a936d39a15f30cc85846dab17c0c7f8b777f974515548c65d4cf3d09a1be0ae1c629afdc0087cef4087389ea6c8f2842a5efbda41537c4a552ec1655a110d197eb0fed4a8dc9166ba82d3b2cb84600b1a3595e1345394e28065977491b4879aa35ada6ff2e78d6408f6e9a69b709503645e954d79432567058c90e26428a4f89d490061e4596c7b9daf5d7850b7d615920978f758b4df8422300e659444ccc045ad182e2a2e0e79b06b717f882147e9e2e3ef8e2939ada17332bb0feec1f0e5bea6de4416c6b70bf9f0056aee795bb838da193578c1b7d44a9de845803971783eca4f11e8b1ebb0bb0e503a724db510b604fd3e70356a21f5a65b5e585665976aa6cca5e1b0cfe4b752881beb5ac56e735c9cf11ac769938eb477efc12c66be313e7b7bf0e492733287807a0d18b7b946da8d9dabfd802ef57ba790f74bc9541b8f8ac9cf51ec81947852e0dc9579dbb36ba3eff340eab3b01d15181e20960a7cca5cecd717d903703ffb221c410a3dd018497306b0e85b5be386c93c29fe8275459f788c680d60332cdf62eb252eacc52abe2979ce0958194debfb9c71a724fbf01e491c7f9eab6be02e303fc50c0098a967aa36d4f5d286cce71300b693b664399737139ba4e594a4104be67f8886499d68b1112ca940f89f04d08b82536ee610431e01686060fb51df0ca6488d99f9d2abc4b2eaedf2625589ed9db4aac302d72f222bb3a52925564052be0d58708325177d5f60095c485a6942145ab430e4d0a86a0679748b652c2c97bea1c1c39d2483825bb37bb4364f374170e8a157b3b5390257ade9cea6679cefd6892bb007e936c3621c20f3ad830fb37504c15295ed0688269e4beccd04f780314e1ca09df7288c048973c13fb1653e5393988e724a5a5ff52e752dfeaea56c84729feed01a4a0c3a606aa648a632ec78d36978779961aaabc35f9826bce515cd138a7feda27f589d0c1c0452349e2e085bb6524607723fa1145c96024a1b6a7e9b759ba553bb78ad3c8bc07eb077202e6726ae613a2376d41033a00a8c57c1f3cb9d154bf45ec79ca89adfc94ef12ef846bf67572cdec1ee42cda6a77d886b2f926",
```

Comment:
Can you change that into a single place that we can easily change?
maybe also as a variable to the queries?

Reply:
It actually was on the same file.
but because the "datafile" string of each form is huge, I separated it.
The function on every ./getAll/ file is almost identical, but making it available to attach each call it's types in the future (this Hashavshevet endpoint returns VERY different types of results).
Would you recommand putting all the "datafile"s on one file, and importing them all to a get-functions file?